front/app/components/ParticipationCTABars/VotingCTABar/index.tsx

```diff
@@ -43,123 +36,34 @@ export const VotingCTABar = ({ phases, project }: CTABarProps) => {
   const participationContext = currentPhase ?? project;
   const basketId = participationContext.relationships.user_basket?.data?.id;
 
-  participationContext.attributes.voting_method;
-
   const { data: basket } = useBasket(basketId);
-  const { mutate: updateBasket } = useUpdateBasket();
 
   if (hasProjectEndedOrIsArchived(project, currentPhase)) {
     return null;
   }
 
-  const submittedAt = basket?.data.attributes.submitted_at || null;
-  const hasUserParticipated = !!submittedAt;
-
-  const minVotes = participationContext.attributes.voting_min_total ?? 0;
-  const maxVotes = participationContext.attributes.voting_max_total;
-
-  const minVotesRequired = minVotes > 0;
-  const minVotesReached =
-    numberOfVotesCast !== undefined ? numberOfVotesCast >= minVotes : false;
-  const minVotesRequiredNotReached = minVotesRequired && !minVotesReached;
-  const votesExceedLimit =
-    maxVotes && numberOfVotesCast !== undefined
-      ? numberOfVotesCast > maxVotes
-      : false;
-
-  const handleSubmitOnClick = () => {
-    if (!isNilOrError(basket)) {
-      const update = () => {
-        updateBasket(
-          {
-            id: basket.data.id,
-            submitted: true,
-            participation_context_type: currentPhase ? 'Phase' : 'Project',
-          },
-          {
-            onSuccess: () => {
-              setProcessing(false);
-              confetti.addConfetti();
-              scrollToElement({
-                id: 'voting-status-module',
-              });
-            },
-          }
-        );
-      };
-
-      if (votingProcessing) {
-        // Add a bit of timeout so that the voting request
-        // has time to complete
-        setTimeout(() => {
-          update();
-        }, 300);
-      } else {
-        update();
-      }
-    }
-  };
-
   const votingMethod = participationContext.attributes.voting_method;
   if (!votingMethod || numberOfVotesCast === undefined) return null;
 
-  const disabledMessage = getDisabledMessage(
-    votingMethod,
-    votesExceedLimit,
+  const currency = appConfig?.data.attributes.settings.core.currency;
+
+  const votesCounter = getVotesCounter(
```

Comment:
Also replaced the `VotesCounter` component with this utility function in the CTA bar. In the `VoteInputs` the VotesCounter` has also been removed